### PR TITLE
Taxon constraints update

### PR DESCRIPTION
**Pull request by balhoff**

### CODE REVIEW

File: docs/explanation/taxon-constraints-explainer.md [hunk 1]
```diff
@@ -2,67 +2,307 @@
...
+- If `Superclasses` includes the term of interest (and the query isn't equivalent to `owl:Nothing`), then there is no particular taxon restriction involving that taxon.
+
+### OBO taxon constraints plugin for Protégé
+
+To quickly see exactly which taxon restrictions are in effect for a selected term, install the [OBO taxon constraints plugin for Protégé](https://github.com/geneontology/protege-taxon-constraints). Once you have the plugin installed, you can add it to your Protégé window by going to the menu `Window > Views > OBO views > Taxon constraints`, and then clicking the location to place the panel. The plugin will show the taxon constraints in effect for the selected OWL class. When a reasoner is running, any inferred taxon constraints will be shown along with directly asserted ones. The plugin executes many reasoner queries behind the scenes, so there may be a delay before the user interface is updated.
+
+<img width="1648" alt="image" src="https://user-images.githubusercontent.com/210210/211460355-262c637e-dfd7-4dc3-b301-04d80ee47ddf.png">
+
+## How to add taxon restrictions:
```
Comment (shawntanzk) [inline docs/explanation/taxon-constraints-explainer.md:306]: Could I suggest we move this up to before `## Using taxon restrictions for Quality Control` - I understand the logic of doing it this way, but I'm thinking more on the side of an ontology editor which might get intimidated by the new additions. Would like to funnel them into the how-to before that happens and all.

**Review by shawntanzk — CHANGES_REQUESTED** [review]
This is awesome! love the mermaid graphs :D Just added a suggestion to move a section up. Thanks!

**Comment by balhoff** [comment]
I think this is ready to merge.

**Comment by matentzn** [comment]
@shawntanzk I will let you merge this!

**Comment by shawntanzk** [comment]
Looks good, thanks so much @balhoff - learnt so much from this :)

**Pull request merged**